<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FakeESP32 panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>133</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QCheckBox" name="high_inner_pressure_alarm">
        <property name="text">
         <string>High inner pressure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="partial_gas_occlusion_alarm">
        <property name="text">
         <string>Partial gas circuit occlusion</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="low_inner_pressure_alarm">
        <property name="text">
         <string>Low inner pressure</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="battery_low_alarm">
        <property name="text">
         <string>Low battery</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="high_input_pressure_alarm">
        <property name="text">
         <string>High pressure in</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="gas_occlusion_alarm">
        <property name="text">
         <string>Gas circuit occlusion</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="low_input_pressure_alarm">
        <property name="text">
         <string>Low pressure in</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="gas_leakage_alarm">
        <property name="text">
         <string>Gas circuit leakage</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="apnea_alarm">
        <property name="text">
         <string>Apnea alarm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="system_failure_alarm">
        <property name="text">
         <string>System failure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="o2_warning">
        <property name="text">
         <string>O2 calib needed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="power_warning">
        <property name="text">
         <string>Power disconnection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="raise_warning_btn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>120</y>
       <width>130</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Raise HW warning</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>473</x>
       <y>10</y>
       <width>16</width>
       <height>131</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="raise_alarm_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>130</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Raise HW alarm</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Event log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW Alarms and Warnings</string>
    </property>
   </widget>
   <widget class="QFrame" name="monitor_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>761</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="monitors_grid"/>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monitor data</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="event_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="status_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_change_status">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Status</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="status_run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>124</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="status_mode">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>124</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="status_backup">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>124</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Backup</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
